src/breadcrumb/Breadcrumb.tsx

```diff
@@ -10,13 +10,22 @@ export type CrumbProps = Readonly<{
   url: string
 }>
 
+export enum DividerPosition {
+  TOP = 'top',
+  BOTTOM = 'bottom',
+  NONE = 'none',
+}
+
 export type BreadcrumbProps = {
   crumbs: Array<CrumbProps>
+  divider?: DividerPosition
 }
 
-export const Breadcrumb = ({ crumbs }: BreadcrumbProps) => (
+export const Breadcrumb = ({ crumbs, divider }: BreadcrumbProps) => (
```

Comment:
The render function still a function and you can sort the elements by composing instead of repeating yourself.
Maybe you should try something like:

```suggestion
const dividerElm = <ContentDivider />
const breadcrumElm = <StyledBreadcrumb divider={divider} itemScope itemType="https://schema.org/BreadcrumbList" />

return (
<Fragment>{divider === DividerPosition.TOP ? [dividerElm, breadcrumElm] : [breadcrumElm, dividerElm]}</Fragment>
)
```

Reply:
I disagree, I'd rather keep JSX rendering declarative, it's easier to read than building it programatically.

Reply:
I don't have a strong opinion, I'd say in this case there's not much complexity so I prefer to read it as a whole in the render. Also there's not just TOP or BOTTOM but also NONE it's not just a ternary condition.